.travis.yml

```diff
@@ -39,57 +41,86 @@ matrix:
 
         # Try MacOS X
         - os: osx
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test -V' OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test -V'
 
         # Check for sphinx doc build warnings - we do this first because it
-        # runs for a long time
-        # OPTIONAL_DEPS needed because the plot_directive in sphinx needs them
+        # runs for a long time. All dependencies are needed because the
+        # plot_directive in sphinx needs them
         - os: linux
-          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='pyephem pytest-mpl wcsaxes astroquery'
         - os: linux
-          env: PYTHON_VERSION=3.4 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=3.5 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='pyephem pytest-mpl wcsaxes astroquery'
 
         # Try all python versions with the latest numpy
         # TODO: add the `--open-files` option back once this issue has been resolved:
         # https://github.com/astropy/astroplan/pull/83#issuecomment-136129489
         - os: linux
           env: PYTHON_VERSION=2.7 SETUP_CMD='test -V'
+               CONDA_DEPENDENCIES='pytz'
+               PIP_DEPENDENCIES=''
+
+        # There isn't numpy 1.10 and python 3.3 combination in conda or
+        # astropy-ci-extras, yet. Update this when it becomes available
```

Comment:
Done. However I didn't mean to add a new build, but to change numpy_version to 1.10 when/if this combination becomes available.